### ISSUE TITLE
[A11y] 1.4.1 Use of Colour – Colour Solely Used to Convey Information

### ISSUE THREAD

**Issue by dalonsoa** [research-software-directory/RSD-as-a-service#1732]
See https://github.com/ImperialCollegeLondon/RSD-as-a-service/issues/45

**Comment by ewan-escience** [comment]
I don't know what to to with this one. We don't rely on colour here to convey information, but on the whole shape of the graph.

**Comment by dmijatovic** [comment]
Maybe we can add some aria-label to the chart stating it is "line graph of github activity" if we do not have that already?

**Comment by dalonsoa** [comment]
I would suggest doing as @dmijatovic suggest, but I will also add a comment on the issue for the accessibility testers explaining the situation. I agree that in this case the colour is irrelevant, with the information being in the shape - unless the line cannot be distinguish from the background at all or to support completely blind users. If that were the case, I really don't know how to fix this.

**Comment by dalonsoa** [comment]
Apparently, after reviewing the issue, they have concluded it is not really an issue as there are no multiple traces to compare and therefore using only colour does not cause any confusion to the user. Hence, closing this issue as there's nothing to fix.
